bentoml/_internal/bento/bento.py

```diff
@@ -159,9 +183,63 @@ def push(self):
         pass
 
     def validate(self):
-        return self.fs.isfile("bento.yaml")
+        return self.fs.isfile(BENTO_YAML_FILENAME)
```

Comment:
Probably just put out a warning if `bentoml.__version__ != self.metadata.bentoml_version`, and only raise an error if we break backward compatibility for certain versions

Reply:
We should definitely do that in the `Bento#from_fs` call, will add a TODO note for now

Reply:
in the older version of BentoML, we check base on the semver of BentoML itself, and it will only raise an error if the major version digit is different. I think we can probably continue that strategy